build/common-config.sh

```diff
@@ -96,6 +100,21 @@ var_set() {
 	"
 	env_key_sort[$i]="PDO_ENCLAVE_CODE_SIGN_PEM"; i=$i+1; export PDO_ENCLAVE_CODE_SIGN_PEM=${env_val[PDO_ENCLAVE_CODE_SIGN_PEM]}
 
+    env_val[PDO_IAS_ROOT_CA_CERT_PATH]="${PDO_IAS_ROOT_CA_CERT_PATH:-${PDO_SGX_KEY_ROOT}/ias_root_ca.cert}"
+    env_desc[PDO_IAS_ROOT_CA_CERT_PATH]="
+        The path of the IAS report signing CA certificate. This is the self-signed certificate of IAS, which
+        certifies the attestation verification key.
+    "
+    env_key_sort[$i]="PDO_IAS_ROOT_CA_CERT_PATH"; i=$i+1; export PDO_IAS_ROOT_CA_CERT_PATH=${env_val[PDO_IAS_ROOT_CA_CERT_PATH]}
+
+    env_val[PDO_IAS_SIGNING_CERT_PATH]="${PDO_IAS_SIGNING_CERT_PATH:-${PDO_SGX_KEY_ROOT}/ias_signing.cert}"
+    env_desc[PDO_IAS_SIGNING_CERT_PATH]="
+        The path of the IAS report signing certificate. This certificate contains the attestation
+        verification key. The path of the file containing the PEM format of this key is the value
+        of the PDO_IAS_KEY_PEM variable.
+    "
+    env_key_sort[$i]="PDO_IAS_SIGNING_CERT_PATH"; i=$i+1; export PDO_IAS_SIGNING_CERT_PATH=${env_val[PDO_IAS_SIGNING_CERT_PATH]}
```

Comment:
same as above... specifically, do we really need these as environment variables? or should we just assume they are in the SGX_KEY_ROOT directory (these are the ones we're creating)

Reply:
With these we can avoid hard-coding the path in the EServiceEnclaveInfoCLI.py.
Also, given the default value, I don't think it's necessary to document them.
This said, I am ok if we decide to remove them.